internal/netconf/interfaces.go

```diff
@@ -82,7 +82,12 @@ func NewIfacesConfigApplier(kind BareMetalType, kb KnowledgeBase, tmpFile string
 // Validate network interfaces configuration. Assumes ifupdown2 is available.
 func (v IfacesValidator) Validate() error {
 	log.Infof("running 'ifup --syntax-check --all --interfaces %s to validate changes.'", v.path)
-	return exec.NewVerboseCmd("ifup", "--syntax-check", "--all", "--interfaces", v.path).Run()
+	err := exec.NewVerboseCmd("ifup", "--syntax-check", "--all", "--interfaces", v.path).Run()
```

Comment:
i dont know actually howto copy with this, because it is always called even in the case of systemd-networkd.
Maybe introducing a new type SystemdNetworkdValidator

Reply:
Introducing a SystemdNetworkdValidator for the case of systemd-networkd validation sounds good to me.

Reply:
Done